data.sql

```diff
@@ -45,4 +45,22 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
-UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+/*PROJECT DAY 4 */
+INSERT INTO vets (name, age, date_of_graduation) VALUES 
+  ('William Tatcher', 45, '2000-04-23'), 
+  ('Maisy Smith', 26, '2019-01-17'), 
+  ('Stephanie Mendez', 64, '1981-05-04'), 
+  ('Jack Harkness', 38, '2008-01-08');
+
+  INSERT INTO specializations (vet_id,species_id) VALUES (1,1), (3,1),(4,2);
+
+  INSERT INTO visits (animal_id, vets_id, date_of_visit) VALUES 
+	(1, 1, '2020-05-24'), (1, 2, '2020-07-22'), (2,3,'2021-02-02').
+	(3, 2, '2020-01-05'), (3, 2, '2020-03-08'), (3, 2, '2020-05-14'),
+	(4, 3, '2021-05-04'), (5, 4, '2021-02-24'), (6,2,'2019-11-21'),
+	(6,1,'2020-08-10'), (6,2,'2021-04-07'), (7,3,'2019-09-29'),
+	(8, 4, '2020-10-03'), (8, 4, '2020-11-04'), (9,2,'2019-01-24'),
+	(9,2,'2019-05-15'), (9,2,'2020-02-27'), (9,2,'2020-08-03'),
+	(10, 3, '2020-05-24'), (10, 1, '2021-01-11');
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230603112-a4be5ca6-1046-4b58-aa34-51953c6cb69d.png)

- [ ] Kindly see your queries and compare with them with the queries highlighted in the image above to fix three queries in this insertion 👍